.github/workflows/compatibility-report.yml

```diff
@@ -54,17 +54,59 @@ jobs:
           path: compatibility.md
 
       # Generate the new PR to update the doc in the repo.
-      - run: |
```

Comment:
why was this removed?

Reply:
The `create-pull-request` action actually does this internally. I discovered it while reviewing the documentation for this PR.